slick.groupitemmetadataprovider.js

```diff
@@ -20,9 +20,9 @@
    * @module Data
    * @namespace Slick.Data
    * @constructor
-   * @param options
+   * @param _options
    */
-  function GroupItemMetadataProvider(options) {
+  function GroupItemMetadataProvider(_options) {
```

Comment:
Just a small change, I would prefer to see this one keep the previous naming since anything that starts with an underscore `_myVar` is typically a private variable, while this one is obviously a public variable.

Reply:
If you don't want to conflict with the other variable, then maybe just name it `inputOptions` or anything else that make sense

Reply:
okay, productive feedback, will change the name to `inputOptions` and recommit.

Reply:
`_options` renamed to `inputOptions`